src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__start-day--tue calendar__month-length--28">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day-tue calendar--month-length-28">
```
`__` is used for block elements. `--` is for modifiers
https://css-tricks.com/bem-101/